src/CompatibilityAdapterBuilder.ps1

```diff
@@ -621,7 +621,7 @@ $OutputTransformations
     }
 
     hidden [string] GetParametersDefinitions([PSCustomObject] $Command) {
-        $commonParameterNames = @("Verbose", "Debug","ErrorAction", "ErrorVariable", "WarningAction", "WarningVariable", "OutBuffer", "PipelineVariable", "OutVariable", "InformationAction", "InformationVariable","WhatIf","Confirm")  
+        $commonParameterNames = @("ProgressAction","Verbose", "Debug","ErrorAction", "ErrorVariable", "WarningAction", "WarningVariable", "OutBuffer", "PipelineVariable", "OutVariable", "InformationAction", "InformationVariable","WhatIf","Confirm")  
```

Comment:
If we are adding `ProgressAction`, we need to update all cmdlets with customizations

Reply:
if we remove progressAction from here, facing errors in UT for commandlets which are auto generated by framework,
following is list 

Enable-EntraDirectoryRole
Get-EntraApplication
Get-EntraApplicationModule
Get-EntraDevice
Get-EntraDirectoryRole
Get-EntraGroup
Get-EntraServicePrincipalOwnedObject
Get-EntraUser
New-EntraApplication
New-EntraGroup
Remove-EntraApplication
Remove-EntraDevice
Remove-EntraGroup
Set-EntraApplication
Set-EntraDevice
Set-EntraGroup
Update-EntraSignedInUserPassword